ocfweb/docs/docs/staff/policy.md

```diff
@@ -0,0 +1,83 @@
+[[!meta title="Staff policy"]]
+
+
+Staff members shall be chosen by the OCF Decision Making Process. [per OCF
+Board decision on 4/13/89]
+
+
+## Background
+
+The OCF Decision Making Process spells out the powers of BoD, the GMs, and the
+SMs; however, it doesn’t explain what powers other staffers have. The main
+purpose of this policy is to delineate what things staff can do on their own,
+and what things staff require special permission from the SM to do.
+
+## Authorization
+
+Staff is authorized to act independently of the SMs unless otherwise specified
+here or elsewhere. This is so decisions do not have to be delayed by the
+bureaucracy of having to ask the SMs for approval for all decisions.
+
+## Root Privileges
+
+The Site Managers have the sole power to add staff to the `ocfroot` group,
+otherwise grant root privileges on a machine which runs a production service or
+which mounts NFS, create and grant privileges to `/admin` principals, and grant
+other powers which can be used to gain the aforementioned privileges or access
+other OCF members' data.
```

Comment:
Since this is an SM-established (and SM-amendable) policy, I would on the contrary keep those specifics in this policy to be very clear about what is being referred to. If the technical implementation changes the SM can simply update the policy to reflect the new details. This contrasts with BoD-established policies which require a BoD vote to amend and therefore shouldn't be tied to technical details.